packages/webpack/src/config/server.js

```diff
@@ -95,7 +95,7 @@ export default class WebpackServerConfig extends WebpackBaseConfig {
         maxEntrypointSize: Infinity,
         maxAssetSize: Infinity
       },
-      externals: []
+      externals: [].concat(config.externals || [])
```

Comment:
Can be simplified to `Array.from(config.externals)` ?

Reply:
Array.from('node_modules') is `["n", "o", "d", "e", "_", "m", "o", "d", "u", "l", "e", "s"]`